cms/models.py

```diff
@@ -80,6 +113,42 @@ def save(self, *args, **kwargs):
         self.slug = slugify("bootcamp-{}".format(self.product.klass_key))
         super().save(*args, **kwargs)
 
+class LearningAreasPage(BootcampProductChildPage):
+    """
+    Learning area's page for learning benefits.
+    """
+
+    subpage_types = []
```

Comment:
This should be set on `BootcampProductChildPage` as almost all child pages would be the end leaf nodes.